ipmikvm

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
shellcheck gave lots of errors because `In POSIX sh, 'local' is undefined.`, that's why I initially switched to using bash.

Also, newer tests, like `[[ -v VAR ]]`, does not exist in sh.

But if I should stay with `/bin/sh` I would remove the `local`s and convert my `[[`s to `test`